src/stream/src/common/log_store_impl/kv_log_store/reader.rs

```diff
@@ -404,7 +453,8 @@ impl<S: StateStore> LogReader for KvLogStoreReader<S> {
         Ok(())
     }
 
-    async fn rewind(&mut self) -> LogStoreResult<bool> {
+    async fn rewind(&mut self) -> LogStoreResult<(bool, Option<Bitmap>)> {
```

Comment:
whats the bitmap for?

Reply:
Currently the initial vnode bitmap of sink writer is passed by sink writer param during `new`. Later the sink may update to new vnode bitmap, in which case the latest vnode bitmap will be different to the one in the initial sink writer param. So in rewind we will return the latest vnode bitmap from the log reader, and later when we create the sink writer, we pass the latest vnode bitmap.